app/styles/main.scss

```diff
@@ -23,6 +23,17 @@ html, body {
   height: 100%;
 }
 
+.flexbox-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  min-height: 100vh;
```

Comment:
I think vh is a percent relative to the screen, and % is based on parent container. So maybe %   is more clear in this case?